### PR TITLE
Dan - UI Design Patterns Challenge

### PR DESCRIPTION
**Pull request by dan12411**
- Xcode 12.2 for the UI Design Patterns Challenge.
- 4th language in (Chinese, Traditional).

Looking forward your feedback, thanks!

### CODE REVIEW

File: MVP Module/Source/Feed UI/Controllers/FeedRefreshViewController.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import UIKit
+import FeedFeature
+
+final class FeedRefreshViewController: NSObject {
+	var delegate: FeedViewControllerDelegate?
+	@IBOutlet var view: UIRefreshControl?
```
Comment (caiozullo) [inline MVP Module/Source/Feed UI/Controllers/FeedRefreshViewController.swift:6]: Since the `FeedRefreshViewController` manages two views, it'd be best to give both a clear name.

`errorView` is great, but the `view` property can be a bit ambiguous. Maybe rename it to `refreshControl`.

File: MVVM Module/Source/Feed UI/Controllers/FeedRefreshViewController.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import UIKit
+import FeedFeature
+
+final class FeedRefreshViewController: NSObject {
+	@IBOutlet var view: UIRefreshControl?
```
Comment (caiozullo) [inline MVVM Module/Source/Feed UI/Controllers/FeedRefreshViewController.swift:5]: Since the `FeedRefreshViewController` manages two views, it'd be best to give both a clear name.

`errorView` is great, but the `view` property can be a bit ambiguous. Maybe rename it to `refreshControl`.

**Review by caiozullo — COMMENTED** [review]
Hi @dan12411,

Congratulations, you've passed the challenge 🥇

- All tests are passing ✅
- Your code is clean and handles edge cases ✅
- The UI and animations look and behave as expected ✅
- Light and Dark mode support ✅
- Added an extra language ✅

I left just one suggestion. Feel free to push changes.

Great job!

File: MVP Module/Source/Feed UI/Controllers/FeedRefreshViewController.swift (hunk 1 above)
Comment (dan12411) [reply MVP Module/Source/Feed UI/Controllers/FeedRefreshViewController.swift]: Thanks Caio, I have renamed to `refreshControl`.

File: MVVM Module/Source/Feed UI/Controllers/FeedRefreshViewController.swift (hunk 2 above)
Comment (dan12411) [reply MVVM Module/Source/Feed UI/Controllers/FeedRefreshViewController.swift]: Thanks Caio, I have renamed to `refreshControl`.

**Comment by caiozullo** [comment]
Nice one!
